chains/evm/contracts/ccvs/CommitteeRamp.sol

```diff
@@ -38,16 +43,15 @@ contract CommitOnRamp is Ownable2StepMsgSender, BaseOnRamp {
     _setDynamicConfig(dynamicConfig);
   }
 
-  /// @notice Forwards a message from CCV proxy to this verifier for processing and returns verifier-specific data.
-  /// @dev This function is called by the CCV proxy to delegate message verification to this specific verifier.
-  /// It performs critical validation to ensure message integrity and proper sequencing.
-  /// @param message Decoded MessageV1 payload for verification.
-  /// @param 2nd parameter is messageId, unused. Allows the onRamp to use the messageId to enable their offchain components.
-  /// @param 3rd parameter is feeToken, unused. Helps determine verifierReturnData.
-  /// @param 4th parameter is feeTokenAmount, unused. Helps determine verifierReturnData.
-  /// @param 5th parameter, is verifierArgs (opaque verifier-specific args), unused. Helps determine verifierReturnData.
-  /// @return verifierReturnData Verifier-specific encoded data.
+  function supportsInterface(
```

Comment:
nit: we generally put suportsInterface at the bottom of the file as other things are more interesting